scripts/bot-deployment/DeployJsonScheduler.sh

```diff
@@ -0,0 +1,28 @@
+if [ $# -ne 6 ]; then
```

Comment:
this script takes in 6 parameters and infers nothing from the GCP environment. For now, I'm leaving this super general like this as there are a few scenarios that I'm not 100% set on how they will be configured so leaving it general like this is the safest bet while being flexible.

Reply:
Yeah, I think we should really try to bring this down in the future. There isn't much value to this script if it can't infer anything from the GCP environment. But I think this is a good start, and we can make this more sophisticated as we go!

Reply:
Yeah I'd personally want some default values

Reply:
Ye, so the main point of this script is to get rid of needing to know the GCP cloud syntax which is a bit cryptic. especially the `headers` part is non standard and is not documented anywhere in their documentation.

We'll refine this script once we've figured out exactly what the deployment process looks like for this hub and spoke model in the future.